src/main/java/eu/dissco/core/translator/terms/TermMapper.java

```diff
@@ -160,23 +23,11 @@ public String retrieveFromDWCA(Term term, JsonNode unit) {
         return value.asText();
       }
     }
-    return term.retrieveFromDWCA(unit);
-  }
-
-  public String retrieveFromABCD(Term term, JsonNode unit) {
-    var termName = term.getTerm();
-    if (mapping.getDefaultMappings().containsKey(termName)) {
-      return mapping.getDefaultMappings().get(termName);
-    } else if (mapping.getFieldMappings().containsKey(termName)) {
-      var value = unit.get(mapping.getFieldMappings().get(termName));
-      if (value != null && value.isTextual()) {
```

Comment:
:+1: Not an exception as it could be empty. Say we put in an explicit fieldmapping for preparation but for half the specimen this hasn't been recorded. Still valid it will just be left empty. I added a log.info message.